Dockerfile

```diff
@@ -32,11 +32,17 @@ RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 RUN useradd -m -s /bin/bash appuser
 WORKDIR /app
 
-# Copy only the necessary release binaries
+# Copy the necessary release binaries
 COPY --from=builder /usr/src/app/target/release/refactor_platform_rs .
-COPY --from=builder /usr/src/app/target/release/migration .
+# Naming this to avoid collision with the migration directory added below
+COPY --from=builder /usr/src/app/target/release/migration migration-binary
```

Comment:
@lmcdonough @jhodapp note that this probably has implications for how we run migrations in production `migration-binary` vs `migration`

Reply:
Updated in e7233f073cccd932925bcc1cc87c07f845b0decf